### PR TITLE
Prepared changelog for KKP 2.25.0 release

### PR DESCRIPTION
**Pull request by archups**
**What this PR does / why we need it**:
This is the provisional changelog for March KKP release 2.25.0. 
Generated changelog until commit ba29d93fdc0b3426aa599f90cc9b851c1c1df25a

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Comment by xrstf** [comment]
@archups Didn't we release beta.0 already?

**Comment by archups** [comment]
> @archups Didn't we release beta.0 already?

@xrstf this was the provisional changelog we had prepared earlier. I have now updated it further for KKP 2.25.0 release.

File: docs/changelogs/CHANGELOG-2.25.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+- **ACTION REQUIRED:** [User MLA] Cortex chart upgraded to resolve issues for cortex-compactor and improve stability of user-cluster MLA feature. Few actions are required to be taken to use new upgraded charts ([#12935](https://github.com/kubermatic/kubermatic/pull/12935)):
+    - Refer to [Upstream helm chart values](https://github.com/cortexproject/cortex-helm-chart/blob/v2.1.0/values.yaml) to see the latest default values.
+    - Some of the values from earlier `values.yaml` are now incompatible with latest version. They are removed in the `values.yaml` in the current chart. But if you had copied the original values.yaml to customize it further, you may see that `kubermatic-installer` will detect such incompatible options and churn out errors and explain that action that needs to be taken.
+    - The memcached-* charts are now subcharts of cortex chart so if you provided configuration for `memcached-*` blocks in your `values.yaml` for user-mla, you must move them under `cortex:` block.
+
```
Comment (csengerszabo) [inline docs/changelogs/CHANGELOG-2.25.md:26]: Let's create a Highlights section here as well including:
- EE: Add KubeVirt ([#12851](https://github.com/kubermatic/kubermatic/pull/12851))
- Upstream Documentation and SourceURLs can be added to ApplicationDefinitions ([#13019](https://github.com/kubermatic/kubermatic/pull/13019))
- EE: Add k8sgpt operator to the Default Application Catalogue ([#13025](https://github.com/kubermatic/kubermatic/pull/13025))
- Add K8sGPT to the Webshell ([#6501](https://github.com/kubermatic/dashboard/pull/6501))
- Add new feature to create, restore and schedule backups for user cluster namespaces ([#6296](https://github.com/kubermatic/dashboard/pull/6296))
- Add new page to manage backup storage location for the cluster backup feature ([#6478](https://github.com/kubermatic/dashboard/pull/6478))
- Support for downloading backups from the UI ([#6521](https://github.com/kubermatic/dashboard/pull/6521))
- Add support for Edge provider ([#6502](https://github.com/kubermatic/dashboard/pull/6502))
- Display comments in application values ([#6510](https://github.com/kubermatic/dashboard/pull/6510))
- Add Support for Kubernetes 1.29 ([#12936](https://github.com/kubermatic/kubermatic/pull/12936))

File: docs/changelogs/CHANGELOG-2.25.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+### Documentation
+
+- Examples now include command to generate secrets that works on vanilla macOS ([#12974](https://github.com/kubermatic/kubermatic/pull/12974))
+
+### Miscellaneous
+
+- Addon manifests are now loaded once upon startup of the seed-controller-manager instead of during every reconciliation. Invalid addons will now send the seed-controller-manager into a crash loop ([#12684](https://github.com/kubermatic/kubermatic/pull/12684))
+- Kube state metrics can be configured to get metrics for custom kubernetes resources ([#13027](https://github.com/kubermatic/kubermatic/pull/13027))
+
+### Dashboard & API
```
Comment (csengerszabo) [inline docs/changelogs/CHANGELOG-2.25.md:182]: Is this empty or should be one level higher?

**Review by csengerszabo — CHANGES_REQUESTED** [review]

File: docs/changelogs/CHANGELOG-2.25.md (hunk 1 above)
Comment (archups) [reply docs/changelogs/CHANGELOG-2.25.md:26]: done

File: docs/changelogs/CHANGELOG-2.25.md (hunk 2 above)
Comment (archups) [reply docs/changelogs/CHANGELOG-2.25.md:182]: Fixed the formatting

**Review by csengerszabo — APPROVED** [review]
/lgtm

**Comment by xrstf** [comment]
/lgtm cancel

Release isn't finished yet, no need to LGTM the changelog already.

File: docs/changelogs/CHANGELOG-2.25.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+- **ACTION REQUIRED:** [User MLA] Cortex chart upgraded to resolve issues for cortex-compactor and improve stability of user-cluster MLA feature. Few actions are required to be taken to use new upgraded charts ([#12935](https://github.com/kubermatic/kubermatic/pull/12935)):
+    - Refer to [Upstream helm chart values](https://github.com/cortexproject/cortex-helm-chart/blob/v2.1.0/values.yaml) to see the latest default values.
+    - Some of the values from earlier `values.yaml` are now incompatible with latest version. They are removed in the `values.yaml` in the current chart. But if you had copied the original values.yaml to customize it further, you may see that `kubermatic-installer` will detect such incompatible options and churn out errors and explain that action that needs to be taken.
+    - The memcached-* charts are now subcharts of cortex chart so if you provided configuration for `memcached-*` blocks in your `values.yaml` for user-mla, you must move them under `cortex:` block.
+
+### Highlights
+- EE: Add KubeVirt ([#12851](https://github.com/kubermatic/kubermatic/pull/12851))
```
Comment (embik) [inline docs/changelogs/CHANGELOG-2.25.md]: This doesn't make enough sense on its own, can we update it to

```suggestion
- EE: Add KubeVirt to the Default Application Catalogue ([#12851](https://github.com/kubermatic/kubermatic/pull/12851))
```

File: docs/changelogs/CHANGELOG-2.25.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+#### Supported Versions
+
+- 1.27.10
+- 1.27.11
+- 1.28.6
+- 1.28.7 (default for k8s)
+- 1.29.1
+- 1.29.2
+- 1.29 (default for EKS and AKS)
```
Comment (embik) [inline docs/changelogs/CHANGELOG-2.25.md]: No need to list this, historically this section is only for native user clusters. So this line can be removed, and the `(default for k8s)` note can also go.

File: docs/changelogs/CHANGELOG-2.25.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+- **ACTION REQUIRED:** For velero helm chart upgrade related change. If you use `velero.restic.deploy: true`, you will see new daemonset `node-agent` running in `velero` namespace. You might need to remove existing daemonset named `restic` manually ([#12998](https://github.com/kubermatic/kubermatic/pull/12998))
+- **ACTION REQUIRED:** For velero helm chart upgrade. If running node-agent daemonset along with velero, then following replacement should be made in the velero's values.yaml before proceeding with upgrade `velero.restic.deploy` with `velero.deployNodeAgent``velero.restic.resources` with `velero.nodeAgent.resources``velero.restic.nodeSelector` with `velero.nodeAgent.nodeSelector``velero.restic.affinity` with `velero.restic.affinity``velero.restic.tolerations` with `velero.nodeAgent.tolerations` ([#13118](https://github.com/kubermatic/kubermatic/pull/13118))
```
Comment (embik) [inline docs/changelogs/CHANGELOG-2.25.md]:
```suggestion
- **ACTION REQUIRED:** For velero helm chart upgrade. If running node-agent daemonset along with velero, then following replacement should be made in the velero's values.yaml before proceeding with upgrade ([#13118](https://github.com/kubermatic/kubermatic/pull/13118))
  - `velero.restic.deploy` with `velero.deployNodeAgent`
  - `velero.restic.resources` with `velero.nodeAgent.resources`
  - `velero.restic.nodeSelector` with `velero.nodeAgent.nodeSelector`
  - `velero.restic.affinity` with `velero.nodeAgent.affinity`
  - `velero.restic.tolerations` with `velero.nodeAgent.tolerations` 
```

**Review by embik — CHANGES_REQUESTED** [review]
some smaller changes required, overall this is looking great!

**Review by embik — APPROVED** [review]
/approve

**Pull request merged**